planet/src/main/FirebaseSecurityRules.txt

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains the security rules of the firebase realtime databse
+{
+    "rules": {
+        // Rules applied to the users tree
+        // Users can read other users' data and write their own data
+        "users": {
+            "$uid": {
+                ".read": true,
+                ".write": "$uid === auth.uid"
+            }
+        },
+        
+        // Rules applied to the places tree
+        // Users can read all places data, but can only edit their own data
+        "places": {
+          	".read": true,
+            "$placeId":{
```

Comment:
Same for the placeId tree. We need to add ".read: true," to be able to read all the list of visitors at once.

Reply:
It's already true from its parent node :)

Reply:
Okay!